skorch/callbacks/logging.py

```diff
@@ -91,10 +99,14 @@ def initialize(self):
 
     def format_row(self, row, key, color):
         """For a given row from the table, format it (i.e. floating
-        points and color if applicable.
+        points and color if applicable).
 
         """
         value = row[key]
+
+        if isinstance(value, bool) or value is None:
```

Comment:
If I see it correctly, this also modifies the output of values that are not events. I'm not sure if we really want this.

Furthermore, given the change above, I'd like to suggest changing the implementation in a way that "falsey" values lead to `''` and "truthy" values to `'+'`:

```
event_symbols = {True: '+', False: ''}
value = event_symbols[bool(value)]
```

If we have that, we could even move `event_symbols` to the class level, making it easier for the user the change the symbols if needed.